handlers.py

```diff
@@ -1,9 +1,19 @@
 import kopf
 import kubernetes
-
-
-@kopf.on.create('', 'v1', 'secrets', annotations={'synator/sync': 'yes'})
-@kopf.on.update('', 'v1', 'secrets', annotations={'synator/sync': 'yes'})
+import os
+
+def watch_namespace(namespace, **_):
+    namespaces = os.getenv('WATCH_NAMESPACE')
+    if namespaces is None or namespaces == "":
+        return True
+
+    all_namespaces = namespaces.split(',')
```

Comment:
Could you set this variable out of function? We don't need to parse namespaces every time

Reply:
Can you please verify?